CalibCalorimetry/EcalTrivialCondModules/src/EcalTrivialConditionRetriever.cc

```diff
@@ -3271,3 +3292,77 @@ EcalTrivialConditionRetriever::produceEcalSamplesCorrelation( const EcalSamplesC
        back_inserter(ipar->EEG1SamplesCorrelation));
   return ipar;
 }
+
+
+
+
+
+
+
+
+std::unique_ptr<EcalSimPulseShape>
+EcalTrivialConditionRetriever::getEcalSimPulseShapeFromConfiguration 
+( const EcalSimPulseShapeRcd& )
+{
+  std::cout<< "Produce EcalSimPulseShape set time interval="<<sim_pulse_shape_TI<< " ns"<< std::endl; 
```

Comment:
`cout` is not allowed, remove it everywhere (or use MessageLogger with appropriate severity level)

Reply:
done